docs/topics/mta-cli-transform.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-cli-transform_{context}"]
+= Performing transformation using the command line
+
+Transform has two sub commands - `openrewrite` and `rules`.
+
+[source,terminal,subs="attributes+"]
+----
+Transform application source code or windup XML rules
+
+Usage:
+  windup-cli transform [flags]
+  windup-cli transform [command]
+
+Available Commands:
+  openrewrite Transform application source code using OpenRewrite recipes
+  rules       Convert XML rules to YAML
+
+Flags:
+  -h, --help   help for transform
+
+Global Flags:
+      --log-level int   log level (default 5)
+
+Use "windup-cli transform [command] --help" for more information about a command.
+----
+
+`OpenRewrite`
+
+The `openrewrite` sub command allows running `OpenRewrite` recipes on source code.
+
+[source,terminal,subs="attributes+"]
+----
+Transform application source code using OpenRewrite recipes
+
+Usage:
+  windup-cli transform openrewrite [flags]
+
+Flags:
+  -g, --goal string     target goal (default "dryRun")
+  -h, --help            help for openrewrite
+  -i, --input string    path to application source code directory
+  -l, --list-targets    list all available OpenRewrite recipes
+  -t, --target string   target openrewrite recipe to use. Run --list-targets to get a list of packaged recipes.
+
+Global Flags:
+      --log-level int   log level (default 5)
+----
+
+To run `transform openrewrite` on application source code, run:
+
+[source,terminal,subs="attributes+"]
+----
+windup-cli transform openrewrite --input=<path/to/source/code> --target=<exactly_one_target_from_the_list>
+----
+
+`Rules`
+
+The `rules` sub command allows converting Windup XML rules to analyzer-lsp YAML rules using `windup-shim`.
+
+----
+Convert XML rules to YAML
+
+Usage:
+  windup-cli transform rules [flags]
+
+Flags:
+  -h, --help                help for rules
+  -i, --input stringArray   path to XML rule file(s) or directory
+  -o, --output string       path to output directory
+
+Global Flags:
+      --log-level int   log level (default 5)
```

Comment:
The log level is an uint32 and defaults to 4

There is an extra global flag:

     --no-cleanup         do not cleanup temporary resources